src/Time/Units.hs

```diff
@@ -118,32 +120,73 @@ instance KnownSymbol (ShowUnit unit) => Show (Time unit) where
 -- Creation helpers
 ----------------------------------------------------------------------------
 
+-- | Creates 'Time' of some type from given 'Natural'.
+time :: Natural -> Time unit
+time n = Time (n :% 1)
+{-# INLINE time #-}
+
+-- | Creates 'Second' from given 'Natural'.
+-- >>> sec 42
+-- 42s :: Second
 sec :: Natural -> Second
-sec n = Time (n % 1)
+sec = time
+{-# INLINE sec #-}
 
+-- | Creates 'MilliSecond' from given 'Natural'.
+-- >>> ms 42
+-- 42ms :: MilliSecond
 ms :: Natural -> MilliSecond
-ms n = Time (n % 1)
+ms = time
+{-# INLINE ms #-}
 
+-- | Creates 'MicroSecond' from given 'Natural'.
+-- >>> mcs 42
+-- 42mcs :: MicroSecond
 mcs :: Natural -> MicroSecond
-mcs n = Time (n % 1)
+mcs = time
+{-# INLINE mcs #-}
 
+-- | Creates 'NanoSecond' from given 'Natural'.
+-- >>> ns 42
+-- 42ns :: NanoSecond
 ns :: Natural -> NanoSecond
-ns n = Time (n % 1)
+ns = time
+{-# INLINE ns #-}
 
+-- | Creates 'Minute' from given 'Natural'.
+-- >>> minute 42
+-- 42m :: Minute
 minute :: Natural -> Minute
-minute n = Time (n % 1)
+minute = time
+{-# INLINE minute #-}
 
+-- | Creates 'Hour' from given 'Natural'.
+-- >>> hour 42
+-- 42h :: Hour
 hour :: Natural -> Hour
-hour n = Time (n % 1)
+hour = time
+{-# INLINE hour #-}
 
+-- | Creates 'Day' from given 'Natural'.
+-- >>> day 42
+-- 42d :: Day
 day :: Natural -> Day
-day n = Time (n % 1)
+day = time
+{-# INLINE day #-}
 
+-- | Creates 'Week' from given 'Natural'.
+-- >>> sec 42
+-- 42w :: Week
 week :: Natural -> Week
-week n = Time (n % 1)
+week = time
+{-# INLINE week #-}
 
+-- | Creates 'Fortnight' from given 'Natural'.
+-- >>> fortnight 42
+-- 42fn :: Fortnight
 fortnight :: Natural -> Fortnight
-fortnight n = Time (n % 1)
+fortnight = time
```

Comment:
You don't need to test all these functions. But you can now use them in tests to make tests nicer :slightly_smiling_face: 
This is also some sort of testing.